src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt

```diff
@@ -80,6 +80,10 @@ object VarMap {
                 mc.networkHandler?.getPlayerListEntry(it.uuid)?.latency?.toDouble()
             } ?: -1.0
         },
+        "serverbrand" string { mc.player?.serverBrand.toString() },
+        "servername" string { mc.currentServerEntry?.name ?: "Singleplayer"},
+        "serverip" string { mc.currentServerEntry?.address ?: "" },
+        "serverversion" string { mc.currentServerEntry?.version?.string ?: mc.gameVersion ?: "" },
```

Comment:
these should be using an `_` to separate the words, similar to `fps_fast` which is above

Reply:
Ah shit yeah that makes sense.

Reply:
Also the default server ip if null should just be "Offline" or something

Reply:
👍